src/Faker/Provider/es_PE/Person.php

```diff
@@ -25,9 +25,9 @@ class Person extends \Faker\Provider\Person
     ];
 
     protected static $firstNameMale = [
-        'Aarón', 'Adrián', 'Agustín', 'Alan', 'Alejandro', 'Alex', 'Alexander', 'Alonso', 'Andrés', 'Anthony', 'Antonio', 'Axel', 'Benjamín',
```

Comment:
Like other providers can we list each first letter on a new line for these lists here?

This will help in tracking changes.

I also miss a @see reference as mentioned in the guide. Where is the data coming from?

Reply:
Quick heads-up, @fzaninotto would usually reject pull requests modifying data sets, because they could break tests down-stream.

Here is an example: https://github.com/fzaninotto/Faker/pull/1775#issuecomment-525027301.

I have no objection to modifying data sets.

Reply:
I agree with the fact that changing datasets is fine.

Reply:
Same here but would like to see a source instead of random names that someone likes.